test/SlowTests/Issues/RavenDB-21427.cs

```diff
@@ -0,0 +1,476 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using FastTests.Utils;
+using Raven.Client.Documents;
+using Raven.Client.Documents.Indexes;
+using Raven.Client.Documents.Operations.Backups;
+using Raven.Client.Documents.Operations.Configuration;
+using Raven.Client.Documents.Operations.ConnectionStrings;
+using Raven.Client.Documents.Operations.DataArchival;
+using Raven.Client.Documents.Operations.ETL.Queue;
+using Raven.Client.Documents.Operations.Expiration;
+using Raven.Client.Documents.Operations.Indexes;
+using Raven.Client.Documents.Operations.QueueSink;
+using Raven.Client.Documents.Operations.Refresh;
+using Raven.Client.Documents.Operations.Revisions;
+using Raven.Client.Documents.Queries.Sorting;
+using Raven.Client.Exceptions.Commercial;
+using Raven.Client.ServerWide;
+using Raven.Client.ServerWide.Operations.Configuration;
+using Raven.Client.ServerWide.Sharding;
+using Raven.Client.Util;
+using Raven.Server;
+using Raven.Server.Commercial;
+using Raven.Server.Documents.Indexes;
+using Raven.Server.Documents.Indexes.Auto;
+using Raven.Server.ServerWide.Commands;
+using Tests.Infrastructure;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace SlowTests.Issues
+{
+    public class RavenDB_21427 : ReplicationTestBase
```

Comment:
All of the tests added here have been skipped on Jenkins? Is it intentional?

Reply:
We'll deploy the relevant licenses next week